acceptance/indy_acceptance/indy-tests/libraries/common.py

```diff
@@ -0,0 +1,159 @@
+
+        :param pool_name: Name of the pool ledger configuration.
+        :param pool_genesis_txn_file: Pool configuration json. if NULL, then default config will be used.
+        :return: The pool handle was created.
+        """
+        import os
+        if os.path.exists(pool_genesis_txn_file) is not True:
+            error_message = Colors.FAIL + "\n{}\n".format(Message.ERR_PATH_DOES_NOT_EXIST.format(Constant.pool_genesis_txn_file)) + Colors.ENDC
+            raise ValueError(error_message)
+
+        print(Colors.HEADER + "\nCreate Ledger\n" + Colors.ENDC)
+        pool_config = json.dumps({"genesis_txn": str(pool_genesis_txn_file)})
+        await pool.create_pool_ledger_config(pool_name, pool_config)
+
+        print(Colors.HEADER + "\nOpen pool ledger\n" + Colors.ENDC)
+        pool_handle = await pool.open_pool_ledger(pool_name, None)
+        return pool_handle
+
+    @staticmethod
+    async def create_and_open_wallet(pool_name, wallet_name):
+        """
+        Creates a new secure wallet with the given unique name.
+        Then open that wallet and get the wallet handle that can
+        be used later to use in methods that require wallet access.
+
+        :param pool_name: Name of the pool that corresponds to this wallet.
+        :param wallet_name: Name of the wallet.
+        :return: The wallet handle was created.
+        """
+        print(Colors.HEADER + "\nCreate wallet\n" + Colors.ENDC)
+        await wallet.create_wallet(pool_name, wallet_name, None, None, None)
+
+        print(Colors.HEADER + "\nGet wallet handle\n" + Colors.ENDC)
+        wallet_handle = await wallet.open_wallet(wallet_name, None, None)
+        return wallet_handle
+
+    @staticmethod
+    async def close_pool_and_wallet(pool_handle, wallet_handle):
+        """
+        Close the pool and wallet with the pool and wallet handle.
+
+        :param pool_handle: pool handle returned by indy_open_pool_ledger.
+        :param wallet_handle: wallet handle returned by indy_open_wallet.
+        :raise Exception if the method has error.
+        """
+        print(Colors.HEADER + "\nClose pool\n" + Colors.ENDC)
+        await pool.close_pool_ledger(pool_handle)
+
+        print(Colors.HEADER + "\nClose wallet\n" + Colors.ENDC)
+        await wallet.close_wallet(wallet_handle)
+
+    @staticmethod
+    async def delete_pool_and_wallet(pool_name, wallet_name):
+        """
+        Delete the pool and wallet with the pool and wallet name.
+
+        :param pool_name: Name of the pool that corresponds to this wallet.
+        :param wallet_name: Name of the wallet to delete.
+        :raise Exception if the method has error.
+        """
+        print(Colors.HEADER + "\nDelete pool\n" + Colors.ENDC)
+        await pool.delete_pool_ledger_config(pool_name)
+
+        print(Colors.HEADER + "\nDelete wallet\n" + Colors.ENDC)
+        await wallet.delete_wallet(wallet_name, None)
+        await asyncio.sleep(0)
```

Comment:
This line seems to be debug stuff and should be removed.

Reply:
Updated. Please refer to the latest code.